devtools/create_project/scripts/postbuild.cmd

```diff
@@ -4,37 +4,36 @@ REM ---------------------------------------------------------------
 REM -- Post-Build Script
 REM ---------------------------------------------------------------
 REM
-REM Copy engine data, themes, translation and required dlls to the
+REM Copy engine data and required dlls to the
 REM output folder and optionally create an installer
 REM
 REM Expected parameters
 REM    Root folder
 REM    Output folder
-REM    Architecture
+REM    SDL version
 REM    Libs folder
 REM    Installer ("1" to build, "0" to skip)
 
 if "%~1"=="" goto error_root
 if "%~2"=="" goto error_output
-if "%~3"=="" goto error_arch
+if "%~3"=="" goto error_sdl
 if "%~4"=="" goto error_libs
 if "%~5"=="" goto error_installer
 
 echo Copying data files
 echo.
 
-xcopy /F /Y "%~4/lib/%~3/SDL.dll"                          "%~2" 1>NUL 2>&1
-xcopy /F /Y "%~4/lib/%~3/freetype6.dll"                    "%~2" 1>NUL 2>&1
+xcopy /F /Y "%~4/%~3.dll"                                  "%~2" 1>NUL 2>&1
+xcopy /F /Y "%~4/WinSparkle.dll"                           "%~2" 1>NUL 2>&1
 xcopy /F /Y "%~1/backends/vkeybd/packs/vkeybd_default.zip" "%~2" 1>NUL 2>&1
 xcopy /F /Y "%~1/backends/vkeybd/packs/vkeybd_small.zip"   "%~2" 1>NUL 2>&1
-xcopy /F /Y "%~1/gui/themes/translations.dat"              "%~2" 1>NUL 2>&1
 

```

Comment:
What is the reason for removing freetype6.dll and translations.dat here?

Reply:
The translations.dat is built into the executable now as part of dists/scummvm.rc, and there's no longer a freetype6.dll in scummvm_libs (seems to be statically-linked now).
The vkeybd_* files might also be obsolete (if necessary they should be added to the .rc) but I couldn't find any references to them so left them alone.